frontend/app/dashboard/components/FeaturedBooks.tsx

```diff
@@ -0,0 +1,19 @@
+import { BookCard } from 'components/cards/BookCard'
```

Comment:
Just want to confirm. Is it recommended by Next.js 13 to add the components inside the apps folder? Instead of in a separate components folder outside apps?

Reply:
Hmmm. Not that I know any restrictions with the current `app` folder. As long as there's a `page.tsx`, the UI for a particular route is defined inside of the folder.

This is how I structured the foldering of this app:
- component folder inside app folder are server components AND closely related components that will work within that page. Mostly these components that I create here are `containers` 
- components at the root of the project are components that I think will be globally available through out the whole app. Think of it as the "unit" of the app, such as buttons, cards, inputs, etc.